### ISSUE TITLE
fullname, name returns empty for types

### ISSUE THREAD

**Issue by usaccounts** [Hookyns/ts-reflection#6]
```
import { getType } from "tst-reflect";

function printTypeProperties<TType>() {
    const type = getType<TType>();
    console.log(type.name)
    console.log(type.fullName)
}

type SomeType = {
    foo: string;
    bar: number;
    baz: Date;
}
printTypeProperties<SomeType>();
```

**Comment by Hookyns** [comment]
Hello @usaccounts thank you for pointing this out.
It is fixed in tst-reflect-transformer@0.8.4

**Comment by usaccounts** [comment]
Cheers, thanks.
